core/message-tags-3.3.md

```diff
@@ -179,6 +201,19 @@ A message sent by a client with the `+example-client-tag` client-only tag:
 
 ---
 
+A message with the same tags sent with both the normal and JSON encodings for comparison:
+
+    @tag=value;+client-tag=value\swith\sspaces PRIVMSG #channel :Message
+    @{"tag":"value","+client-tag":"value\swith\spaces"} PRIVMSG #channel :Message
```

Comment:
So `{"tag":"\" \";"}` becomes `@{"tag":"\\"\s\\"\:"}`? I don't see why it needs to be so complicated and why it need to escape so much.
`@{"tag":"\"\u0020\";"}` looks simpler and is within JSON spec.